src/main/java/com/google/api/codegen/transformer/php/PhpSurfaceNamer.java

```diff
@@ -124,6 +125,11 @@ public String getFullyQualifiedApiWrapperClassName(GapicInterfaceConfig interfac
   }
 
   @Override
+  public String getApiWrapperClassImplName(InterfaceConfig interfaceConfig) {
+    return publicClassName(Name.upperCamel(getInterfaceName(interfaceConfig), "Gapic"));
```

Comment:
Yes, I was uncertain about this. I wanted to avoid `ClientImpl`, but didn't have many better ideas. `GapicClient` maybe?